File: VictorSP.java
//==============================================================================
/**
 * @file    VictorSP.java
 * @date    2/27/2018
 * @author  Cierra
 * @author  Ricardo
 * 
 */
//==============================================================================
/**
 * @class VictorSP
 */
public class VictorSP
{
	
	
	private int motorPin;
	
	VictorSP(int PWM) {
		this.motorPin = PWM;
	}

	public void set(double speed) {
		System.out.format("The speed is %.2f and the pin is %d%n", speed, motorPin);
	}

	
}

File: CrossTheLineCpp/VictorSP.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   VictorSP.cpp
 * Author: Cierra
 * 
 * Created on March 2, 2018, 6:53 PM
 */

#include <iostream>
#include "VictorSP.h"
    
    VictorSP::VictorSP(int PWM) {
        this->motorPin = PWM;
    }
    
    void VictorSP::set(float speed) {
        std::cout << "The speed is " << speed << std::endl;
    }

    VictorSP::~VictorSP() {
    }

